packages/integration/src/azure/core.ts

```diff
@@ -100,7 +100,17 @@ export function getAzureDownloadUrl(url: string): string {
     ? `&scopePath=${encodeURIComponent(filepath)}`
     : '';
 
-  return `${protocol}://${resource}/${organization}/${project}/_apis/git/repositories/${repoName}/items?recursionLevel=full&download=true&api-version=6.0${scopePath}`;
+  if (url.includes('dev.azure.com')) {
```

Comment:
```suggestion
  if (resource === 'dev.azure.com') {
```

this is probably what you want to do here. That also gets rid of the CodeQL build error.

Reply:
Thanks, I've made the change you suggested